libs/utils/reports.py

```diff
@@ -207,17 +215,23 @@ def peak_hour_violations(self, camera_id):
     def average_violations(self, camera_id):
         log_dir = os.getenv('LogDirectory')
         file_path = os.path.join(log_dir, camera_id, "objects_log", "report.csv")
+        if not os.path.exists(file_path):
+            return 0.0
         df = pd.read_csv(file_path).drop(['Number'], axis=1)
         return round(df['ViolatingObjects'].mean(), 1)
 
     def camera_with_most_violations(self):
         log_dir = os.getenv('LogDirectory')
         cameras = os.listdir(log_dir)
```

Comment:
I don't think this is the correct way to check for a list of cameras. You should get them from the config file. There could be other files and folders in there, not just `.keep`

Reply:
The issue is that this file doesn't have access to the configuration (since we can't deliver the object to the reports subapi). Maybe we could add the .keep to the docker ignore?

Reply:
Hm no, that doesn't work because data is a mounted volume